client/src/services.ts

```diff
@@ -1,34 +1,38 @@
-import { EmployeeI } from "./types";
+import { IEmployee } from "./types";
 
 const url = "http://localhost:3000/api";
 
-export const addEmployee = async (employee: EmployeeI): Promise<boolean> => {
-  let added = false;
+export const addEmployee = async (employee: IEmployee): Promise<boolean> => {
   try {
-    const { statusText } = await fetch(`${url}/add-employee`, {
+    const { status } = await fetch(`${url}/employee`, {
       method: "POST",
       body: JSON.stringify(employee),
     });
-    if (statusText === "OK") added = true;
+    if (status === 200) {
```

Comment:
It's better to use constants like const OK_STATUS_CODE = 200, because if (status === OK_STATUS_CODE ) more readable than if (status === 200)

Reply:
Changed